library/Core/src/Juvix/Core/IR/CheckTerm.hs

```diff
@@ -375,6 +375,21 @@ requireStar ::
 requireStar (IR.VStar j) = pure j
 requireStar ty = Error.throwTC (Error.ShouldBeStar ty)
 
+-- | Given the arity of a type, require all of them to be Star. We expect the
+-- arguments to be written in a right-associative way, e.g: @* -> (* -> *)@.
+-- TODO: check that pi == 0.
```

Comment:
Stale TODO?
```suggestion
```

Reply:
That is the `requireZero pi` one. Guess I forgot to remove the TODO.